docs/UserGuide.md

```diff
@@ -157,6 +170,7 @@ Format: `find FOOD`
 * Only the calorie contents of some food items are included. Refer to the list of food items [here](#food-items).
 * If the user has entered a specific food item before, the `find` command will return the average calories of 
 that specific food item that the user has specified.
+* This applies to food items that have been entered on previous days, and not just on the current day. 
```

Comment:
I think it would be better to change line 168 to:

Format: `find FOOD_NAME`

So that it is consistent with the other commands.